dataclients/kubernetes/definitions/testdata/validation/route-with-multiple-filters-one-item.log

```diff
@@ -1 +1 @@
-single filter expected
+single filter expected at "foo(42) -> bar(24)"
```

Comment:
minor: add newline here for :stop_sign: at the end of this file to disappear.

Reply:
It's like this because I still didn't make tests work.